pandas/_libs/hashing.pyx

```diff
@@ -162,6 +159,8 @@ cdef uint64_t low_level_siphash(uint8_t* data, size_t datalen,
     cdef uint8_t* end = data + datalen - (datalen % sizeof(uint64_t))
     cdef int left = datalen & 7
     cdef int left_byte
+    cdef int cROUNDS = 2
+    cdef int dROUNDS = 4
```

Comment:
is there any upside to assigning this at the module level?

Reply:
I don't see any in this case.